include/picongpu/param/physicalConstants.param

```diff
@@ -46,18 +46,22 @@ namespace picongpu
 
         // Electron properties
         /** unit: kg */
-        constexpr float_64 ELECTRON_MASS_SI = 9.109382e-31;
+        constexpr float_64 ELECTRON_MASS_SI = 9.1093837015e-31;
```

Comment:
feel free to push the precision change in a separate PR before you atomic PR.

Reply:
will do!